src/views/view-company-information.njk

```diff
@@ -0,0 +1,41 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/breadcrumbs/macro.njk' import govukBreadcrumbs %}
+{% from 'govuk/components/button/macro.njk'      import govukButton %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
```

Comment:
Don't copy the landing page. Instead, just create a placeholder page, like what Matesuz did for the Search Company page originally (just a title) - https://github.com/companieshouse/dissolution-web/blob/master/src/views/search-company.njk#L22